bridges/centralized-ethereum/Cargo.toml

```diff
@@ -22,5 +22,6 @@ witnet_data_structures = { path = "../../data_structures" }
 witnet_config = { path = "../../config" }
 witnet_net = { path = "../../net" }
 witnet_node = { path = "../../node" }
+witnet_rad = { path = "../../rad" }
```

Comment:
Why is the `current_active_wips` under the rad module?
Maybe it should be under `validations` :)